.travis.yml

```diff
@@ -1,22 +1,70 @@
 language: rust
-sudo: true
```

Comment:
removed by mistake maybe? I think kcov installation would fail without sudo which would mean no code coverage.

Reply:
No, I removed it to test as builds are generally faster on the non-sudo infra. I'll poke around to see if it is needed or not.